decidim-meetings/spec/system/live_meeting_access_spec.rb

```diff
@@ -315,6 +315,40 @@ def visit_meeting
     end
   end
 
+  describe "when a meeting link is avaliable signed in" do
+    let!(:meeting) { create(:meeting, :published, :signed_in_iframe_access_level, :online, component:) }
+
+    context "when user is signed in" do
+      before do
+        login_as user, scope: :user
+      end
+
+      it "shown to signed in users" do
+        visit_meeting
+
+        expect(page).to have_css(".address__hints")
+      end
+    end
```

Comment:
There's actually missing the use case from the original bug report: what happens when the user is not signed in? 

I think it should be like this 
```suggestion
    context "when user is not signed in" do
      it "not shown to not signed in users" do
        visit_meeting

        expect(page).not_to have_css(".address__hints")
      end
    end

    context "when user is signed in" do
      before do
        login_as user, scope: :user
      end

      it "shown to signed in users" do
        visit_meeting

        expect(page).to have_css(".address__hints")
      end
    end
```

And the same context should be applied to the other example that you add

Reply:
@andreslucena and here also if you could test if your changes work as I think its now breaking my CI.